src/Schema/Events/Tap.ts

```diff
@@ -33,6 +33,29 @@ export interface TappedArticleGroup extends TappedEntityGroup {
   action: ActionType.tappedArticleGroup
 }
 
+/**
+ * A user taps a grouping of news on App
+ *
+ *  This schema describes events sent to Segment from [[tappedNewsGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedNewsGroup",
```

Comment:
**question**: Don't we already have a `tappedArticleGroup` event we could use?

Reply:
hmm, that's true - it seems like news are articles 😄

Reply:
I will update the payload to reflect the type of the section (rail / list)